src/toil/batchSystems/gridengine.py

```diff
@@ -222,7 +223,8 @@ def supportsHotDeployment(cls):
 
     def __init__(self, config, maxCores, maxMemory, maxDisk):
         super(GridengineBatchSystem, self).__init__(config, maxCores, maxMemory, maxDisk)
-        self.gridengineResultsFile = self._getResultsFileName(config.jobStore)
+        self.gridengineResultsFile = re.sub('file:', './',
```

Comment:
This would substitute anywhere in the string. 

And it wouldn't work anyways. What if the URL is `file:///foo/bar`, or `file://./foo`? You need to apply the same logic that the file system job store uses to get the path from the URL.